### PR TITLE
Add expires documentation

### PR DESCRIPTION
**Pull request by hlidotbe**
Hi,

I've just implemented a small directive/handler plugin to add expiration headers to the response.
So here's the PR for the doc.

### CODE REVIEW

**Comment by mholt** [comment]
Thanks for the docs! Will merge when the other PR is merged.

File: content/docs/expires.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+title: expires
+type: docs
+directive: true
+plugin: true
+link: https://github.com/epicagency/caddy-expires
+---
+
+expires allows you to set expiration header to resources
```
Comment (mholt) [inline content/docs/expires.md]: Period at the end of the line ;)

File: content/docs/expires.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+It allows you to set different expiration durations depending on a matching regular expression.
+
+### Syntax
+
+<code class="block"><span class="hl-directive">expires</span> {
+	<span class="hl-subdirective">match</span> <i>pattern</i> <i>duration</i>
+}</code>
+
+*   **match** describe a pattern and a expiration duration
```
Comment (mholt) [inline content/docs/expires.md]: "describes" -- make sure to clarify it's a regular expression. Period at end of sentence.

File: content/docs/expires.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+Match subdirective can be repeated as many times as you want but only the first matching will be used
+
+Duration is a combination of 0h0m0d0h0i0s in that order. Parts can be omitted
```
Comment (mholt) [inline content/docs/expires.md]: This does not seem to quite match the code. y/m/d/h/i/s instead? Also, period at end of line. :wink: Check the other lines below for punctuation.

File: content/docs/expires.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    js files after 30 days</span>
+    <span class="hl-subdirective">match</span> <i>\.png$</i> <i>1d</i><span class="hl-comment"># expires
+    png files after one day</span>
+    <span class="hl-subdirective">match</span> <i>\.jpg$</i> <i>1h</i><span class="hl-comment"># expires
+    jpg files after one hour</span>
+    <span class="hl-subdirective">match</span> <i>\.pdf$</i> <i>1i</i><span class="hl-comment"># expires
+    pdf file after one minute</span>
+    <span class="hl-subdirective">match</span> <i>\.txt$</i> <i>1s</i><span class="hl-comment"># expires
+    txt files after one second</span>
+    <span class="hl-subdirective">match</span> <i>\.html$<i></</i>i> 5i30s<span class="hl-comment"># expires
```
Comment (mholt) [inline content/docs/expires.md]: Broken HTML here. I don't generally like the use of italics in code blocks... can we just leave the text straight up and down here? :)

File: content/docs/expires.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+title: expires
+type: docs
+directive: true
+plugin: true
+link: https://github.com/epicagency/caddy-expires
+---
+
+expires allows you to set expiration header to resources
+
+It allows you to set different expiration durations depending on a matching regular expression.
```
Comment (mholt) [inline content/docs/expires.md]: Clarify what it matches on. The path in the URL?

**Comment by hlidotbe** [comment]
I've cleaned up and clarified some parts. Is it better?

Is there a way to preview the docs? I've tried to use hugo to generate the site but the docs gives me an error

**Comment by mholt** [comment]
What error? Just run `hugo` and it will gen the site.

**Comment by mholt** [comment]
I'll merge once you've had a chance to review the page yourself in the browser with hugo and approve!

**Comment by hlidotbe** [comment]
ok got it. there's no index.html generated in docs (at least not for me) so /docs fails. But going directly to /docs/expires.html gives me the correct page. Seems good to me.

**Pull request merged**